lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1499,6 +1513,32 @@ data ErrNoSuchEpoch = ErrNoSuchEpoch
     , errCurrentEpoch :: W.EpochNo
     } deriving (Eq, Show)
 
+getNetworkClock :: Handler ApiNetworkClock
+getNetworkClock = do
+    ntpS <- liftIO $ withNtpClient nullTracer ntpSettings $ \client -> do
+        link $ ntpThread client
+        ntpQueryBlocking client
```

Comment:
All the ntp client state will be destroyed at the end of the `withNtpClient` block. Would it be better to set up the ntp client outside of the servant handler?

Reply:
And have it in the "global" API context like the network layer? Yes, this sounds like a sensible thing to do.

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1396/commits/a35a7d09c5dafd542f6a42d3e445f0fe8d9648d3